script_business_requirement_category/README.rst

```diff
@@ -0,0 +1,48 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
```

Comment:
This is very specific to the current situation: I think this part is not necessary in this repo.